packages/formation/sass/modules/_m-promo-banner.scss

```diff
@@ -0,0 +1,54 @@
+.vads-c-promo-banner {
+  z-index: $base-layer + 1;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+
+  background-color: $color-gray-lightest;
+  color: $color-link-default;
+  font-weight: 700;
+
+  .vads-c-promo-banner__body {
+    display: table;
+    width: 100%;
+  }
+
+  .vads-c-promo-banner__content, va-promo-banner-close {
```

Comment:
looks like a typo here BTW. 

Should be:
```
 .vads-c-promo-banner__content, 
.vads-c-promo-banner__close {
```

Reply:
Haha yep, you caught that right after I wondered why my close button didn't look right on small screens